addons/mail/static/tests/composer/suggested_recipients.test.js

```diff
@@ -223,6 +223,7 @@ test("suggest recipient on 'Send message' composer (all checked by default)", as
     registerArchs(archs);
     await start();
     await openFormView("res.fake", fakeId);
+    await contains(".o-mail-Followers-counter", { text: "0" });
```

Comment:
These asserts will pass immediately without checking anything (0 is the initial state even before doing any action).

For the test to cover something, it should have at least 1 follower initially.